src/components/CreateJob.tsx

```diff
@@ -56,21 +63,23 @@ interface State {
 
 export const createSearchCriteria = (): SearchCriteria => ({
   cloudCover: 10,
-  dateFrom:   moment().subtract(30, 'days').format('YYYY-MM-DD'),
-  dateTo:     moment().format('YYYY-MM-DD'),
-  source:     SOURCE_RAPIDEYE,
+  dateFrom: moment.utc().subtract(30, 'days').format(DATE_FORMAT),
```

Comment:
Should this be UTC?  (No, I really don't know)

Reply:
You're right on this.  For setting the date it should use the date from the local time.

Reply:
I'm making the change.